flink/src/test/java/org/apache/sedona/flink/AggregatorTest.java

```diff
@@ -55,6 +55,14 @@ public void testKNN() {
         assertEquals(5.656854249492381, last(resultTable).getField(0));
     }
 
+    @Test
+    public void testIntersection_Aggr(){
+        Table polygonTable = createPolygonOverlappingTable(testDataSize);
+        Table result = polygonTable.select(call("ST_Intersection_Aggr", $(polygonColNames[0])));
+        Row last = last(result);
+        assertEquals("LINESTRING EMPTY", last.getField(0).toString());
```

Comment:
Please come up with a test case that generates a non-empty result. @yyy1000

Reply:
Done.